lighthouse-core/test/results/sample_v2.json

```diff
@@ -581,6 +581,53 @@
       "scoreDisplayMode": "binary",
       "explanation": ""
     },
+    "unsized-images": {
+      "id": "unsized-images",
+      "title": "Image elements do not have explicit `width` and `height`",
+      "description": "Always include explicit width and height on image elements to reduce layout shifts and improve CLS. [Learn more](https://web.dev/optimize-cls/#images-without-dimensions)",
+      "score": 0,
+      "scoreDisplayMode": "binary",
+      "details": {
+        "type": "table",
+        "headings": [
+          {
+            "key": "url",
+            "itemType": "thumbnail",
+            "text": ""
+          },
+          {
+            "key": "url",
+            "itemType": "url",
+            "text": "URL"
+          },
+          {
+            "key": "node",
+            "itemType": "node",
+            "text": "Failing Elements"
+          }
+        ],
+        "items": [
+          {
+            "url": "http://localhost:10200/dobetterweb/lighthouse-480x318.jpg",
```

Comment:
did we ever add to the ImageElements artifact in the test `artifacts.json` the extra data added?

doesnt seem like it. would be nice to add.

Reply:
yeah I never manually added the extra data, I'll make sure to do so!